### ISSUE TITLE
CKE editor panel style wrong on HD screen (720p)

### ISSUE THREAD

**Issue by memsenpai** [ckeditor/ckeditor-sdk#239]
i found this issue, on scheen 720, the panel is skewed to the right, it does not fit within the object
On the full screen it is still inside.
![image](https://user-images.githubusercontent.com/10399540/36708787-57ccf4da-1ba7-11e8-8323-46aeb8e40262.png)

**Comment by mlewand** [comment]
@memsenpai sorry for the delay, but what panel are you referring to? Based on your information we can't say what panel it relates to.

**Comment by mlewand** [comment]
@memsenpai yes, but what SDK sample is it? Is it somewhere here: https://sdk.ckeditor.com/samples/inline.html

We'd appreciate a full screenshot that would help a lot.

**Comment by memsenpai** [comment]
@mlewand sorry but i cant provide a screenshot because my project is private with high security!
but you can see above picutre, border blue is area initialize ckeditor, border black is ckeditor toolbar.
in 1080p, width OK but broken on 720p. (we also know width of ckeditor set by sdk, we dont touch it).
version sdk 4
